integrations/repo_tag_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"io/ioutil"
+	"net/url"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/git"
+	"code.gitea.io/gitea/modules/util"
+	"code.gitea.io/gitea/services/release"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsUserAllowedToControlTag(t *testing.T) {
```

Comment:
I think this is an unit test. It's better to move it to ``models/tags_test.go``

Reply:
I thought that too but the `IsUserInTeams` method calls the database. Do we have a DB mock? Otherwise it's an integration test.

Reply:
unit tests are run using sqlite db